enclone_print/src/build_table_stuff.rs

```diff
@@ -328,17 +328,18 @@ pub fn build_table_stuff(
                 if ctl.gen_opt.nospaces {
                     let chars = s.chars().collect::<Vec<_>>();
                     s.clear();
-                    for (i, &ch) in chars.iter().enumerate() {
-                        if (i == 0 || (i > 0 && chars[i - 1] != '┅')) && ch == '┅' {
-                            s += "[01m[38;5;198m=";
-                        } else if ch == '┅' {
+                    for i in 0..chars.len() {
+                        if (i == 0 || (i > 0 && chars[i - 1] != '┅')) && chars[i] == '┅' {
```

Comment:
I have no strong opinions about this, but using `iter().enumerate()` pattern doesn't seem ergonomic in this case since we index `chars[i-1]` and `chars[i+1]`.

Reply:
It avoids the bounds check overhead on at least some of the lookups.  Granted would be better to break the first and last element items out of the loop and use zip(skip) to avoid the i-1 as well.